backend/tracim_backend/exceptions.py

```diff
@@ -6,6 +6,10 @@ class TracimError(Exception):
     pass
 
 
+class NoValidCollaborativeDocumentEditionSoftware(TracimError):
```

Comment:
exceptions to move to collaborative edition module

Reply:
Exceptions are currently all in same place, do we really change this behaviour ?
I think it make sense for most thing to have separate file, like i try to do but not sure about exception and almost sure something should be done for the specific error case because it make code really complicated to have error code in multiple place.